src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -19,29 +20,22 @@ def create(data,
            max_iterations=0,
            verbose=True):
     """
-    Create a :class:`ObjectDetector` model.
+    Create a :class:`OneShotObjectDetector` model.
 
     Parameters
     ----------
-
     data : SFrame | tc.Image
-        An SFrame that contains all the starter images along with their
-        corresponding labels.
-        If the dataset is a single tc.Image, this parameter is just that image.
-        Every starter image should entirely be just the starter image without
-        any padding.
-        RGB and RGBA images allowed.
+        A single starter image or an SFrame that contains the starter images
+        along with their corresponding labels. These images can be of in RGB
```

Comment:
You want to get rid of the `of`. On the start of the next line I think `or` makes more since than `and`. Also the padding statement should probably be it's own sentence. 

So I would replace the last sentence of this paragraph with `These image(s) can be in either RGB or RGBA format. They should not be padded.`